test_fatoptimizer.py

```diff
@@ -3206,6 +3206,70 @@ def test_str_encode(self):
         self.check_dont_optimize(r'"ab\xff".encode("big5")')
         self.check_dont_optimize(r'"ab\xff".encode("ascii", "backslashreplace")')
 
+    def test_str_lower(self):
+        self.check_optimize(r'"ABC".lower()', '"abc"')
+
+    def test_str_upper(self):
+        self.check_optimize(r'"ABC".upper()', '"ABC"')
+
+    def test_str_capitalize(self):
+        self.check_optimize(r'"abc".capitalize()', '"Abc"')
+
+    def test_str_swapcase(self):
+        self.check_optimize(r'"aBc".swapcase()', '"AbC"')
+
+    def test_str_casefold(self):
+        self.check_optimize(r'"ABC".casefold()', '"abc"')
+        self.check_optimize(r'"ß".casefold()', '"ss"')
```

Comment:
Please fold these case functions into a single test function. One test perf method is enough, fatoptimizer test suite is not the Python test suite!